ocaml/src/tools/lwt_extra2.ml

```diff
@@ -255,3 +255,24 @@ let read_file file =
        assert (read = len');
        Lwt.return ()) >>= fun () ->
   Lwt.return buf
+
+
+let copy_using fd_in fd_out size buffer =
```

Comment:
Hmm, didn't this loop already exist somewhere? (We probably want the code to be shared)